libs/entitlement/src/test/java/org/elasticsearch/entitlement/runtime/policy/entitlements/FilesEntitlementTests.java

```diff
@@ -98,72 +97,70 @@ public void testFileDataRelativeWithEmptyDirectory() {
-    }
 
-    public void testExclusiveParsing() throws Exception {
-        Policy parsedPolicy = new PolicyParser(new ByteArrayInputStream("""
-                    entitlement-module-name:
-                      - files:
-                        - path: /test
-                          mode: read
-                          exclusive: true
-            """.getBytes(StandardCharsets.UTF_8)), "test-policy.yaml", true).parsePolicy();
-        Policy expected = new Policy(
-            "test-policy.yaml",
-            List.of(
-                new Scope(
-                    "entitlement-module-name",
-                    List.of(FilesEntitlement.build(List.of(Map.of("path", "/test", "mode", "read", "exclusive", true))))
-                )
-            )
-        );
-        assertEquals(expected, parsedPolicy);
+        fileData = FileData.ofConfigPathSetting("foo.bar", READ, false);
+        settings = Settings.builder().put("foo.bar", "relative_path").build();
+        assertThat(fileData.resolvePaths(TEST_PATH_LOOKUP).toList(), contains(Path.of("/config/relative_path")));
     }
 
     public void testPathSettingIgnoreUrl() {
-        var fileData = FileData.ofPathSetting("foo.*.bar", READ, true);
+        var fileData = FileData.ofConfigPathSetting("foo.*.bar", READ, true);
         settings = Settings.builder().put("foo.nonurl.bar", "/setting/path").put("foo.url.bar", "https://mysite").build();
         assertThat(fileData.resolvePaths(TEST_PATH_LOOKUP).toList(), contains(Path.of("/setting/path")));
     }
 
-    public void testRelativePathSettingIgnoreUrl() {
-        var fileData = FileData.ofRelativePathSetting("foo.*.bar", CONFIG, READ, true);
-        settings = Settings.builder().put("foo.nonurl.bar", "path").put("foo.url.bar", "https://mysite").build();
-        assertThat(fileData.resolvePaths(TEST_PATH_LOOKUP).toList(), contains(Path.of("/config/path")));
-    }
-
     public void testIgnoreUrlValidation() {
         var e = expectThrows(
             PolicyValidationException.class,
             () -> FilesEntitlement.build(List.of(Map.of("path", "/foo", "mode", "read", "ignore_url", true)))
         );
-        assertThat(e.getMessage(), is("'ignore_url' may only be used with `path_setting` or `relative_path_setting`"));
+        assertThat(e.getMessage(), is("'ignore_url' may only be used with 'config_path_setting'"));
 
         e = expectThrows(
             PolicyValidationException.class,
             () -> FilesEntitlement.build(
                 List.of(Map.of("relative_path", "foo", "relative_to", "config", "mode", "read", "ignore_url", true))
             )
         );
-        assertThat(e.getMessage(), is("'ignore_url' may only be used with `path_setting` or `relative_path_setting`"));
+        assertThat(e.getMessage(), is("'ignore_url' may only be used with 'config_path_setting'"));
+    }
+
+    public void testExclusiveParsing() throws Exception {
```

Comment:
This isn't testing `config_path_setting` so it seems unrelated to the rest of the PR?

Reply:
I moved tests related to path setting around so they are next to each other again. They were split by the exclusive tests.